.github/ISSUE_TEMPLATE/3-plugin-compatiblity.md

```diff
@@ -0,0 +1,27 @@
+---
+name: "💥 Plugin compatibility"
+about: Create a plugin compatibility issue report
+---
+
+### Your checklist for this issue
+
+🚨 Please review the [guidelines for contributing](../blob/master/docs/CONTRIBUTING.md) to this repository.
+
+- [ ] Create an issue on [issues.jenkins-ci.org](https://issues.jenkins-ci.org), that is commonplace to report plugin issues
```

Comment:
This doesn't really make sense to me:
> that is commonplace to report plugin issues

Maybe something like this?
```suggestion
- [ ] Create an issue on [issues.jenkins-ci.org](https://issues.jenkins-ci.org), set the component to the plugin you are reporting it against
```

Reply:
:+1: